examples/messages/src/App.tsx

```diff
@@ -1,12 +1,13 @@
 import React, {useEffect} from 'react';
-import {DEFAULT_API_V2, store} from "aleph-sdk-ts";
+import {DEFAULT_API_V2} from "aleph-sdk-ts/global";
+import {store} from "aleph-sdk-ts";
 import {ethereum} from "aleph-sdk-ts/accounts";
-import {StorageEngine} from "aleph-sdk-ts/messages/message";
+import {ItemType} from "aleph-sdk-ts/messages/message";
```

Comment:
Import namespaces are incorrect (at least on the pinned version). `ItemType` does not exist (yet, on version used). This should work: 
```js
import { DEFAULT_API_V2, store } from "aleph-sdk-ts/";
import { ethereum } from "aleph-sdk-ts/accounts";
import { StorageEngine } from 'aleph-sdk-ts/messages/message';
```